### DISCUSSION TITLE
Game controller support

### DISCUSSION THREAD

**Discussion by cricri-pingouin** [lwcorp/lwmenu#5] [Ideas]
Would it be possible to add game controller support?
Up/down to go through the buttons (like up/down cursor keys currently) and controller button (configurable or at least button 0) to run.
Would be useful to use as a gaming interface (e.g. menu of games/emulators to launch without needing a keyboard/mouse).
Thanks!

**Comment by lwcorp** [comment]
Interesting. I use the built-in GUI of my programming language, and didn't deliberately block such access, so I guess it's not native.
The thing is, if I try to get it working, how will I know it worked? Would you or someone reading this be willing to donate a game controller, even on a loan?
